---
layout: post
title: Open Reading Frame Disruption in the Archaeal Virus SSV1
category: [DNA Analysis]
tags: [electrophoresis, S. solfataricus, DNA analysis]
---
## Summary

Sulfolobus solfataricus is an archaeal thermophile. It lives in extreme conditions where the temperature climbs higher than 80 degrees C and the pH drops below 3.0. This archaeal species is infected by Sulfolobus spindle-shaped virus 1 (SSV1), a member of the fusellovirus family. Scientists study SSV1 because they are interested in how viruses replicate in extreme conditions. Viral enzymes which operate in these conditions may have applications in science and industry. This study screened SSV1 genomes which contained random insertions of the Tn5 transposon to discover which open reading frame had been mutated, then tested them for infectivity with S. solfataricus. The purpose was to characterize the open reading frames of SSV1 as essential or nonessential for infectivity and compare with past research.  

We received a patch plate containing samples of Escherichia coli which had been transformed with mutant SSV1 DNA containing the Tn5 transposon. We chose samples C3-C7 to work with for this experiment. DNA from samples C3-C7 was isolated by alkaline lysis and GeneJet kit preps and digested with restriction enzymes HindIII, EcoRI, and EcoRV to determine location of the transposon insertion. DNA fragments were isolated on agarose gels and visualized under UV light. Samples C3, C6, and C7 were sequenced at OHSU’s Core laboratory then analyzed with FinchTV and BLAST. Samples C3, C6, and C7 were also transformed into S. solfataricus using electroporation and tested for infectivity with halo assays.

We found that C3 had an insertion between the open reading frames B-251 and D- 335 and seemed to be infectious, C6 had an insertion within A-100 and also seemed to be infectious, and C7 had an insertion within C-792 and was not infectious.

## Methods

E. coli cells transformed with SSV1::Tn5 were grown in LB media with Kanamycin (50 micrograms/ml). The SSV1::Tn5 plasmid was extracted through alkaline lysis (modified from Kratt et al., Biotechniques) and GeneJet kit (Fisher) preps. Three digests were performed with HindIII, EcoRV, and both EcoRI and EcoRV Fast Digest restriction endonucleases and 10x Fast Digest Green Buffer (Fisher). Agarose gel electrophoresis (0.7%, 1%, and 1% agarose respectively) was used to isolate the DNA fragments, which were then visualized under UV light. NEBcutter (nc2.neb.com) was used to predict banding patterns after gel electrophoresis. DNA samples were prepared for sequencing with a BigDyeâ Terminator (Fisher) sequencing kit and sequenced at OHSU’s Core laboratory. Sequence data was analyzed with FinchTV and BLAST. Growth media (modified from Zillig) was prepared for S. solfataricus, which was incubated and then made electrocompetent by treatment with ice-cold sucrose. DNA from samples C3, C6, and C7 was used to electroporate S. solfataricus using a BTX-Gemini machine (Schleper et al., 1992, modified by Iverson) using EAI377 as the positive control. Screening for virus functionality was done with halo assays (modified from Stedman et al., 1999). The concentration of Triton X-100 was reduced by 10x for the second assay. Cultures were incubated at 37°C after spotting.

## Results

The first digest with HindIII showed band disruption for the 3458 bp fragment in sample C3, none for sample C4, two small bands for sample C5, and faint bands for samples C6 and C7 (Figure 1). The second digest with EcoRV was difficult to read, but showed multiple bands for sample C3, two bands for sample C5, and none for samples C6 and C7 (Figure 2). The third digest with EcoRI and EcoRV showed very clear bands for samples C3, C6, and C7 (Figure 3). Sample C3 showed band displacement for the 1468 bp fragment after digestion with EcoRV, sample C6 showed band displacement for the 2695 bp fragment in the EcoRV digestion, and sample C7 showed band displacement for the 2952 bp fragment after EcoRI digestion and band displacement for the 2695 bp fragment after EcoRV digestion. Incubated S. solfataricus samples after electroporation with SSV1 DNA exhibited growth for sample C7 and lack of growth for samples C3 and C6 (Table 1). No halos were observed on either halo assay for DNA samples or controls. Lawn growth and clearing for Triton-X detergent was present on the first assay, but neither were prominent on the second.

## Discussion

The first restriction digest indicated transposon insertion between ORFs B-251 and E-96 in sample C3. Unfortunately, the second digest did not allow further refinement in our predictions. The third digest narrowed the insertion in C3 to between B-251 and D-335. These predictions were verified with sequencing, which indicated that the insertion took place directly between B-251 and D-335. A previous insertion in this location yielded an infectious virus (Iverson et al., 2017) and S. solfataricus electroporated with sample C3 did not exhibit growth in our study, which confirms initial results. Samples C4 and C5 indicated presence of the transposon only. Sample C6 appeared to have an insertion between E-178 and A-132 after the final digest, and sequencing verified that the insertion was within A-100. Previous deletions of A-100 or insertions at its origin, which is also the origin of transcript six, yielded a non-infectious virus (Iverson et al., 2017). Surprisingly, S. solfataricus electroporated with sample C6 did not exhibit growth in our study after both trials, indicating that insertions within A-100 may not be as detrimental to infectivity as insertions at its origin. Perhaps insertions at the origin of A-100 interrupt the reading of transcript six while insertions within it do not. Sample C7 appeared to have an insertion between C-166 and B-251 or between E-178 and A-132, depending on the digest. Sequencing, however, contradicted these predictions and placed the insertion within C-792. This open reading frame is highly conserved and codes for VP4, a minor capsid protein (Iverson et al., 2017). Previous insertions within C-792 yielded a non-infectious virus (Iverson et al., 2017), and we indeed found that S. solfataricus electroporated with sample C7 exhibited growth in our study. (Table 2) Unfortunately, halo assays did not perform as expected, so further testing is needed to confirm results. Possible sources of error may have included overheating S. solfataricus during plating of halo assays and preparation error during DNA isolation or digestion. Insertions between B-251 and D-335 (sample C3) and within C-792 (sample C7) behaved as predicted in past research, while an insertion within A-100 (sample C6) produced an infectious virus, providing a direction for future research, as past deletions of A-100 or insertions near its origin have rendered it noninfectious.

![EcoRV Digest](https://github.com/benjamin-e-moore/Moore_Lab_Notebook/blob/master/images/BMCS_NoLadder.jpg)

Figure 1: Digest of mutant SSV1 with HindIII

![Gel Picture](https://github.com/benjamin-e-moore/Moore_Lab_Notebook/blob/master/images/BCWP%20EcoRV%20Digest%20SSV1.JPG)

Figure 2: Digest of mutant SSV1 with EcoRV

![Gel Picture](https://github.com/benjamin-e-moore/Moore_Lab_Notebook/blob/master/images/CSBM.PNG)

Figure 3: Digest of mutant SSV1 with EcoRI and EcoRV
